### ISSUE TITLE
value proposition vs. current market?

### ISSUE THREAD

**Issue by dckc** [rchain-community/daw205#6]
The current market leaders are services such as sovrin. It's permissioned rather than decentralized, but at this point, so is RChain (cf https://github.com/rchain-community/rstake/issues/6 ).

Common cases of [writing to the Sovrin Public Ledger](https://sovrin.org/issue-credentials/) are free: DIDs for Individuals (Peer DIDs) and Credential Issuance. In RChain, every deploy costs REV. (There is discussion of [sponsored data APIs](https://github.com/rchain/hackathon-2020/issues/21) but not much work has been done.)

sovrin uses technology from what seems to be the most popular self-sovreign identity API, [hyperledger indy](https://github.com/hyperledger/indy-sdk).

[Zero Knowledge Proofs in the Sovrin Network](https://sovrin.org/the-sovrin-network-and-zero-knowledge-proofs/) are deployed in production. Perhaps the same essential design could work with RChain, but it's not clear what benefit justifies the effort / risk.

[Hyperledger members and governance](https://en.wikipedia.org/wiki/Hyperledger#Members_and_governance) shows involvement of ConsenSys, IBM, Intel and so on; [sidetree](https://github.com/decentralized-identity/sidetree) have has involvement from Microsoft.

RChain doesn't even get mentioned in the context of events such as [IIW 31 Oct 2020](https://mattr.global/highlights-of-iiw31/).

RChain's architecture has advantages over these designs in that rholang supports scalable cooperation without vulnerability thru [capability security](https://github.com/dckc/awesome-ocap/wiki/Rholang). But the Digital Accreditation Wallet design presented Nov 18 makes no use of that. (See also [rchain\-community/liquid\-democracy](https://github.com/rchain-community/liquid-democracy))

The value proposition of this design is entirely unclear, to me.

**Comment by dckc** [comment]
more market context:

[The Dangers of Blockchain\-Enabled “Immunity Passports” for COVID\-19 \| by Elizabeth M\. Renieris \| Berkman Klein Center Collection \| Medium](https://medium.com/berkman-klein-center/the-dangers-of-blockchain-enabled-immunity-passports-for-covid-19-5ff84cacb290) May 18

**Comment by dckc** [comment]
more market context:

[Digital Identity on the Blockchain With Chainlink](https://blog.chain.link/digital-identity-on-the-blockchain/) 9 Nov 2020
 - DECO
 - Bloom
 - Unstoppable Domains (as non-fungible ERC721 tokens on Ethereum)
 - Decentr - a Web3 version of credit scores

**Comment by dckc** [comment]
more market context:

[Decentralized Key Management \(DKMS\): An Essential Missing Piece of the SSI Puzzle – Drummond Reed \- YouTube](https://www.youtube.com/watch?v=NJxaYsAo6pI) Jul 2018

In this case, the Digital Accreditation Wallet design presented Nov 18 does seem roughly on par, though this presentation reflects substantial usability research and it's not clear whether we could take advantage of that or if we'd be starting from scratch.

**Comment by TheoXD** [comment]
I have to agree that there is plenty of competition that's trying to do the same thing with much bigger budgets, and the only leverage we have over them is object capability model.
A lot of those alternatives don't force end user to go to an exchange and purchase tokens in order to manage their identity wallet, especially if that token is not allowed to be purchased by US citizens. What we can do however (on the Casper level) is to enable some party to pay for those deploys upfront if we know how much phlo is needed and validators have to respect them as any other deploy. I usually refer to those as pre-paid deploys.
There are of course privacy concerns, moral concerns, etc. I would prefer if the primary goal is not for mass-testing of people for Covid, as there are many more useful usecases, like "does anyone who's boarding this plane have certain alergies, in which case how many are there" rather than "Sir, you don't have a proof of vaccination for covid and are not allowed to travel, pleace go to the end of the line".

**Comment by dckc** [comment]
Civic is another one I ran across this morning. Its current offerings are largely centralized, but the [whitepaper](https://tokensale.civic.com/CivicTokenSaleWhitePaper.pdf) shows a road to decentralization.

 - [list of partners](https://www.civic.com/partners/)
 - [>$50M market cap](https://coinmarketcap.com/currencies/civic/)

**Comment by dckc** [comment]
decentralized storage + transactions motivates RChain.

_quick note from chat with @leithaus ; here's hoping I find time to elaborate_

**Comment by dckc** [comment]
Another one:
 - [Atala PRISM](https://atalaprism.io/) is a decentralized identity solution built on the Cardano blockchain. It creates a new approach to identity management, where users own their identity and have complete control over how their personal data is used and accessed. Data is shared with other individuals or organizations over secure, private peer-to-peer communication channels. Atala PRISM is designed for businesses, governments, and individuals.

They're hiring a scala dev:
 - [Post \| Feed \| LinkedIn](https://www.linkedin.com/feed/update/urn:li:groupPost:746917-6754521053202149376/)
